src/agent/GRPCClientAgent.ts

```diff
@@ -88,14 +88,17 @@ class GRPCClientAgent extends GRPCClient<AgentServiceClient> {
 
   @ready(new agentErrors.ErrorAgentClientDestroyed())
   public vaultsGitPackGet(...args) {
-    return this.client.vaultsGitPackGet(...args);
+    return grpcUtils.promisifyDuplexStreamCall(
+      this.client,
+      this.client.vaultsGitPackGet,
+    )(...args);
   }
 
   @ready(new agentErrors.ErrorAgentClientDestroyed())
-  public vaultsScan(...args) {
-    return grpcUtils.promisifyReadableStreamCall<vaultsPB.Vault>(
+  public nodesScan(...args) {
```

Comment:
Move this back to `vaultsScan`. And allow the generic use of gestalt id.

If too difficult for now, then `nodeid` is fine.